frontend/src/features/database/page/DatabasePage.tsx

```diff
@@ -26,6 +26,7 @@ import { DatabaseDataGrid } from '@/features/database/components/DatabaseDataGri
 import { SearchInput, SelectionClearButton } from '@/components';
 import { SortColumn } from 'react-data-grid';
 import { convertValueForColumn } from '@/lib/utils/database-utils';
+import { BETTER_AUTH_SYSTEM_TABLES } from '@insforge/shared-schemas';
```

Comment:
I think we should hidden system built-in or better-auth tables in backend. For frontend, it's better not to aware these details.